src/plugins/log-requests.js

```diff
@@ -19,11 +19,8 @@ export const logRequests = {
       if (isAuthenticated && credentials.user) {
         const { user } = credentials
 
-        if (
-          'preferred_username' in user &&
-          typeof user.preferred_username === 'string'
-        ) {
-          userPrefix = ` [${user.preferred_username}] `
+        if ('oid' in user && typeof user.oid === 'string') {
```

Comment:
```suggestion
        if (!user.oid) {
```

Reply:
Won't work unfortunately. Not a TS project and the types from Hapi is `MergeType<UserCredentials, UserCredentials>` so `oid` isn't included, and we can't extend the interface.

Reply:
Something like this?

Probably needs a few tweaks or an `@type` to override

```js
  /**
   * @param {Request<{ AuthUser: UserProfile }>} request
   * @param {number} responseTime
   */
customRequestCompleteMessage(request, responseTime) {
```